docs/sources/flow/reference/components/otelcol.processor.tail_sampling.md

```diff
@@ -229,6 +230,9 @@ The following arguments are supported:
 Name | Type | Description | Default | Required
 ---- | ---- | ----------- | ------- | --------
 `min_spans` | `number` | Minimum number of spans in a trace. | | yes
+`max_spans` | `number` | Maximum number of spans in a trace. | `0` | no
+
+The `max_spans` can be set to 0 to indicate max spans should not be considered.
```

Comment:
```suggestion
Set `max_spans` to `0`, if you do not want to limit the policy samples based on the maximum number of spans in a trace.
```

The suggestion is an attempt to make the sentence active and be explicitly clear what's happening. I'm not sure if I got the technical meaning correct though.

Also... do we need to adjust the text in line 226 and define the policy samples based on the maximum (upper limit?) number of spans in a trace? Is it an upper limit to the number of spans? What happens if you have more than X spans? Is the trace not sampled if the max is hit (similar to min)?

Reply:
you've got it, if the max spans is not equal to 0 and the number of spans is greater than the max spans then it does not get sampled.